sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/PhpWorkloadsCRUDTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Workloads.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
```

Comment:
Considering that we already used `System.Text.Json`, can we avoid using `Newtonsoft.Json` here?